app/models/template.rb

```diff
@@ -33,12 +33,12 @@ def self.dmptemplate_ids(org)
 
   # Retrieves the most recent version of the template for the specified Org and dmptemplate_id 
   def self.current(org, dmptemplate_id)
```

Comment:
I think that you are correct. The org_id should not be needed but we have bad data coming through in the migration I think. The dmptemplate_id remains the same when a template is customized. I updated the code so that this is no longer the case, but we need to fix the data first before removing org_id from these helper functions.
```sql
mysql> select id, org_id, dmptemplate_id, customization_of, created_at, title from templates where dmptemplate_id = 1;
+-----+--------+----------------+------------------+---------------------+--------------+
| id  | org_id | dmptemplate_id | customization_of | created_at          | title        |
+-----+--------+----------------+------------------+---------------------+--------------+
|   1 |      3 |              1 |                1 | 2016-01-18 12:33:52 | DCC Template |
|   2 |      1 |              1 |             NULL | 2016-01-18 12:33:52 | DCC Template |
|   3 |      5 |              1 |                1 | 2016-01-18 12:33:52 | DCC Template |
|   4 |     13 |              1 |                1 | 2016-01-18 12:33:52 | DCC Template |
| 117 |      1 |              1 |             NULL | 2017-03-20 20:35:45 | DCC Template |
| 118 |      1 |              1 |             NULL | 2017-03-20 20:36:21 | DCC Template |
| 125 |      1 |              1 |             NULL | 2017-03-21 21:50:54 | DCC Template |
| 128 |      1 |              1 |             NULL | 2017-03-27 19:43:53 | DCC Template |
| 129 |      1 |              1 |             NULL | 2017-03-27 19:43:55 | DCC Template |
| 130 |      1 |              1 |             NULL | 2017-03-27 19:44:10 | DCC Template |
| 131 |      1 |              1 |             NULL | 2017-03-27 19:44:12 | DCC Template |
| 141 |      1 |              1 |             NULL | 2017-04-14 19:01:15 | DCC Template |
| 142 |      1 |              1 |             NULL | 2017-04-14 19:01:16 | DCC Template |
+-----+--------+----------------+------------------+---------------------+--------------+
13 rows in set (0.00 sec)
```